### ISSUE TITLE
Static variables from contract class don't work correctly in auxiliary classes

### ISSUE THREAD

**Issue by hurui200320** [neow3j/neow3j#601]
Hi, I'd like to move some helper functions (in contract class which make it longer than 300+ lines) to a separate class so it will look neat and clean. Since those helper functions are specific to that contract class, it will be great if I can just use that static field rather than pass it as a parameter.

Issue: When using `StorageMap` as a static field, it gives `Invalid type for PICKITEM: InteropInterface` error.
Code:
```java
public class TestContract {
    private static final StorageContext CTX = Storage.getStorageContext();
    static StorageMap MAP = CTX.createMap("some_prefix");

    public static void foo(){
        Auxiliary.helper();
    }
}

public class Auxiliary {
    static void helper() {
        TestContract.MAP.get("key");
    }
}
```

**Comment by hurui200320** [comment]
Also a weird thing about static field in helper class.
There are two static field in contract class:
```java
public class TestContract {
    // 0x218bc33597ce7dc32079f593d5ed5766c1ef7764
    static final Hash160 HASH1 = addressToScriptHash("NV5CSGyT6B39fZJ6zw4x6gh1b3C6cpjTm3");
    // 0xf9077f4eec533d175eed50b789b4d8d55dacf5ff
    static final Hash160 HASH2 = addressToScriptHash("NjFMoMSoukNBetDZYPsGKzpLrUA1zgkMNM");

    public static Hash160 foo(){
        return Auxiliary.helper();
    }
}
```
Then no matter I use first or second one, it always gives me the first one:
```java
public class Auxiliary {
    static Hash160 helper() {
//        return TestContract.HASH1;
        return TestContract.HASH2;
    }
}
```
`foo` return: `6477efc16657edd593f57920c37dce9735c38b21`, which is the little endian representation of the HASH1

**Comment by gsmachado** [comment]
as far as I understood by talking to @csmuller, this was partially solved in #624. 🚀

**Comment by csmuller** [comment]
Sorry, i mixed two issues up. This one is solved. Issue #616 is the one that has only partially been tackled by #624.
